scripts/nginx-setup.py

```diff
@@ -163,7 +200,10 @@ def location(route, directives):
 
     # Handled by router/router.py
     location('/%(repo)s/search', ['proxy_pass http://localhost:8000;'])
+    location('/%(repo)s/sorch', ['proxy_pass http://localhost:8000;'])
     location('/%(repo)s/define', ['proxy_pass http://localhost:8000;'])
+    location('/%(repo)s/symbol', ['proxy_pass http://localhost:8000;'])
```

Comment:
This hunk shouldn't be here, I think?

Reply:
Argh, sorry, yes.  I was cherry-picking deltas and there was an additional hunk I wanted to get the text/plain comment from without taking other changes in the patch and I checked out the tip revision, forgetting that I'd explicitly avoided cherry-picking certain revisions.